lambda/test/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "awsProfile": "foodtruckspotters",
+  "bucket": "foodtruckspotters",
+  "imagePathOriginal": "/test/foodtrucktest.jpg",
+  "imagePathResized": "/test/foodtrucktest-123x123.jpg"
+}
```

Comment:
This file looks like it's part of a separate effort. Can you remove it and amend this commit?